### ISSUE TITLE
ParetoRGG.py not found

### ISSUE THREAD

**Issue by devmessias** [martinjm97/DRGG#1]
1.  There is no ParetoRGG.py inside the repo

2.  I tryed  use the code inside https://github.com/martinjm97/DRGG/blob/master/drgg.py 
 The main code create an object named Simulator and call the method adjancency_lists

https://github.com/martinjm97/DRGG/blob/77a937cb96a0915a11f3da625d8184c40ed9d6d0/drgg_model_metrics.py#L11

but inside the Simualtor class exists only the method "adjacency_lists"

https://github.com/martinjm97/DRGG/blob/77a937cb96a0915a11f3da625d8184c40ed9d6d0/drgg.py#L41

This method  does not work properly for some reasons.

**Comment by ssilwa** [comment]
@martinjm97 can you look into this?

**Comment by martinjm97** [comment]
@devmessias thank you very much for bringing up these problems. I fixed the import issues. 

The other problem was that names of vertices and radii were swapped, which meant that the parameters were passed in in the wrong order. It is now fixed as well.
https://github.com/martinjm97/DRGG/blob/77a937cb96a0915a11f3da625d8184c40ed9d6d0/drgg_model_metrics.py#L9-L10

Everything should now be working. Let us know if you have any other issues and thanks again!
